src/ChainSafe.Gaming.Lootboxes.Chainlink/ILootboxService.cs

```diff
@@ -100,5 +120,26 @@ async Task<List<LootboxTypeInfo>> FetchAllLootboxes()
                 .Select(i => new LootboxTypeInfo { TypeId = typeIds[i], Amount = balances[i] })
                 .ToList();
         }
+
+        /// <summary>
+        /// Gets the native currency price to buy a lootbox.
+        /// </summary>
+        /// <returns>The native currency price as a string.</returns>
+        Task<BigInteger> GetPrice();
+
+        /// <summary>
+        /// Sets the native currency price to buy a lootbox, admin/owner only.
+        /// </summary>
+        /// <param name="price">An amount of native currency user needs to pay to get a single lootbox.</param>
+        /// <returns>A task representing the asynchronous operation.</returns>
+        Task SetPrice(BigInteger price);
```

Comment:
Do we need to set the price from the client? Why not just use dashboard?

Reply:
i dont think it's on the dashboard

Reply:
it's being added into the dash, i'll remove it unity side.